micrometer-spring-legacy/src/samples/java/io/micrometer/spring/samples/PrometheusSample.java

```diff
@@ -22,7 +22,11 @@
 @SpringBootApplication(scanBasePackages = "io.micrometer.spring.samples.components")
 @EnableScheduling
 public class PrometheusSample {
-    public static void main(String[] args) {
-        new SpringApplicationBuilder(PrometheusSample.class).profiles("prometheus").run(args);
-    }
+
+
+	public static void main(String[] args) {
+		new SpringApplicationBuilder(PrometheusSample.class).profiles("prometheus").run(args);
+	}
```

Comment:
What changed here?

Reply:
This is reverted. I made the configuration bean in here at first. :)